wp-content/themes/mambaone/single-cpt_lessons.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*Template Name: Single cpt Lessons
+*/
+
+get_header();
+
+$args = array(
+    'post_type' => 'cpt_lessons',
+    'orderby' => 'publish_date',
+    'order' => 'DESC',
+    'posts_per_page' => 3,
+);
+
+$loop = new WP_Query($args);
+?>
+<?php
```

Comment:
If there is only one php line, better write it as `<?php if ($loop->have_posts()): ?>`
Same for `endif` on lines 30-31